app/helpers/job_applications_helper.rb

```diff
@@ -52,31 +52,30 @@ def job_application_jobseeker_profile_info(job_application)
   end
 
   def job_application_support_needed_info(job_application)
-    case job_application.support_needed
-    when "yes"
+    case job_application.is_support_needed
+    when true
```

Comment:
Should this be turned into an `if/else` as done with `job_application_safeguarding_issues_info` ?

Reply:
no - sadly there are explicit tests for this being nil and producing blank